src/eduid/webapp/personal_data/views.py

```diff
@@ -82,6 +86,65 @@ def update_personal_data(
     personal_data = personal_data_user.to_dict()
     return success_response(payload=personal_data, message=PDataMsg.save_success)
```

Comment:
I think we should add a deprecation decorator to update_personal_data, see https://github.com/SUNET/eduid-backend/blob/2bea72ff3641a546ebb87f23da6b58d8f45d19d7/src/eduid/webapp/authn/views.py#L68